interface/wx/grid.h

```diff
@@ -4336,6 +4356,8 @@ class wxGrid : public wxScrolledWindow
     */
     bool InsertRows(int pos = 0, int numRows = 1, bool updateLabels = true);
 
+    virtual void Refresh(bool eraseb = true, const wxRect* rect = NULL);
```

Comment:
Technically, it isn't needed for documentation purposes.  For Phoenix's purposes, it is better if overridden methods are present in the interface headers.  However, @vadz doesn't agree with me, so I'll remove it.